collections/_article/dev-snapshot-godot-4-2-dev-6.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Dev snapshot: Godot 4.2 dev 6"
+excerpt: "Final 4.2 dev snapshot before we enter the beta phase, and it's absolutely feature-packed!"
+categories: ["pre-release"]
+author: Rémi Verschelde
+image: /storage/blog/covers/dev-snapshot-godot-4-2-dev-6.webp
+image_caption_title: "Restaurant Bits"
+image_caption_description: "An asset pack by Kay Lousberg"
+date: 2023-10-03 22:00:00
+---
+
+This is the final dev snapshot for 4.2, signaling the end of the feature development cycle, and the start of the beta phase. We spent the past couple of weeks wrapping up many of the big feature PRs which had been in development for the past few weeks and months. We still have a handful that the production team is keeping tabs on, but we'll soon declare the full feature freeze for 4.2 and get ready for the first beta snapshot. But don't wait for the beta to try what's new in 4.2 – there are tons of amazing new features in this dev snapshot which require testing and user feedback, so we can iron out the main issues before the stable release.
+
+Here's a selection of some of the biggest changes since our previous [4.2 dev 5 build](/article/dev-snapshot-godot-4-2-dev-5/), ahead of a more complete list [below](#whats-new):
+
+* The rendering contributors have outdone themselves this cycle, and a lot of their work has been merged for this snapshot:
+
+  - AMD's <abbr title="FidelityFX Super Resolution">FSR</abbr> 2.2 has been implemented as a new upscaling option in the project settings. Have fun testing it and let us know how it goes :)
+
+  - An optional ANGLE-backed OpenGL renderer was added for macOS and Windows ([GH-72831](https://github.com/godotengine/godot/pull/72831)). ANGLE is a compatibility layer for OpenGL on top of Metal and Direct3D 11, which allows us to work around the deprecated and unmaintained OpenGL drivers on macOS, and similarly outdated OpenGL drivers on Windows for some older integrated chipsets. This should increase the portability of Godot games on lower end devices.
+
+  - Talking about the OpenGL Compatibility renderer, one of its main missing features, 3D shadows, has now been implemented ([GH-77496](https://github.com/godotengine/godot/pull/77496))!
+
+  - For lightmapping, we replaced the extremely bulky and slow <abbr title="OpenImage Denoise">OIDN</abbr> denoiser with a lightweight and much faster <abbr title="Joint Non-Local Means">JNLM</abbr> denoiser compute shader ([GH-81659](https://github.com/godotengine/godot/pull/81659)). There is a noticeable decrease in denoising quality from the much simpler JNLM approach, but we expect that the results might be satisfactory for most games. Please try it out and let us know if you're happy with the results. If there's demand for it, we might re-introduce OIDN as an option, using it as a standalone command line tool instead of building it together with Godot. With the built-in OIDN removed, editor binaries are now approximately 4-5 MB smaller.
```

Comment:
```suggestion
  - For lightmapping, we replaced the extremely bulky and slow <abbr title="OpenImage Denoise">OIDN</abbr> denoiser with a lightweight and much faster <abbr title="Joint Non-Local Means">JNLM</abbr> denoiser compute shader ([GH-81659](https://github.com/godotengine/godot/pull/81659)). There is a noticeable decrease in denoising quality with the much simpler JNLM approach, but we expect that the results might be satisfactory for most games. Please try it out and let us know if you're happy with the results. If there's demand for it, we might re-introduce OIDN as an option, using it as a standalone command line tool instead of building it together with Godot. With the built-in OIDN removed, editor binaries are now approximately 4-5 MB smaller.
```
"From" could sound like it's referring to the older method.

Reply:
Thanks, fixed.